doc/api/dgram.md

```diff
@@ -225,6 +225,20 @@ never have reason to call this.
 If `multicastInterface` is not specified, the operating system will attempt to
 drop membership on all valid interfaces.
 
+### socket.getRecvBufferSize(size)
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns {number} the `SO_RCVBUF` socket receive buffer size in bytes.
```

Comment:
Just a thought, but we might not need to mention `SO_RCVBUF` and `SO_SNDBUF` at all. They aren't really needed to work with the API.